packages/engine-formula/src/engine/dependency/formula-dependency.ts

```diff
@@ -911,7 +912,7 @@ export class FormulaDependencyGenerator extends Disposable {
                 continue;
             }
 
-            const cacheStack: FormulaDependencyTree[] = [];
+            cacheStack.length = 0;
```

Comment:
comments for this line, it will clear the array.

Reply:
Done